app/file.go

```diff
@@ -1348,7 +1348,7 @@ func (a *App) ExtractContentFromFileInfo(fileInfo *model.FileInfo) error {
 // GetLastAccessibleFileTime returns CreateAt time(from cache) of the last accessible post as per the cloud limit
 func (a *App) GetLastAccessibleFileTime() (int64, *model.AppError) {
 	license := a.Srv().License()
-	if license == nil || !*license.Features.Cloud {
```

Comment:
The method also nil checks itself, so if we wanted we could do `if !a.Srv().License().IsCloud() {`

e.g. this snippet is ok and doesn't blow up:

```
package main

import (
	"fmt"
)

type License struct {
	Features *Features
}

type Features struct {
	Cloud *bool
}

func (l *License) IsCloud() bool {
	return l != nil && l.Features != nil && l.Features.Cloud != nil && *l.Features.Cloud
}

func main() {
	var l *License
	fmt.Printf("is cloud: %v\n", l.IsCloud())
	fmt.Println("OK")

}
```